opm/parser/eclipse/CMakeLists.txt

```diff
@@ -271,7 +272,7 @@ EclipseState/Tables/RtempvdTable.hpp
 EclipseState/Tables/OilvisctTable.hpp
 EclipseState/Tables/GasvisctTable.hpp
 EclipseState/Tables/WatvisctTable.hpp
-EclipseState/Tables/PvtgOuterTable.hpp
+EclipseState/Tables/PvtgTable.hpp
```

Comment:
> but currently there is no PvtoTable class anymore

Oh there is - both `PvtoTable`and `PvtgTable` - the common code in `PvtxTable` is *very* limited.